accounts/accounts_test.go

```diff
@@ -128,7 +128,7 @@ func TestSignRace(t *testing.T) {
 	if err := am.TimedUnlock(a1.Address, "", 15*time.Millisecond); err != nil {
 		t.Fatalf("could not unlock the test account", err)
 	}
-	end := time.Now().Add(80 * time.Millisecond)
+	end := time.Now().Add(500 * time.Millisecond)
 	for time.Now().Before(end) {
```

Comment:
It should yield anyway. See http://golang.org/doc/go1.2#preemption.
I cannot reproduce this failure. Does inserting the call to `Gosched` fix the test for you?